src/components/Callout/Callout.Props.ts

```diff
@@ -81,3 +95,8 @@ export interface ILink {
    */
   url: string;
 }
+
+export enum CalloutBackgroundColor {
+  white,
+  blue
```

Comment:
It depends on what you want customers to choose.

Callout I believe should have no background coloring other than body background color. Classes that wrap the callout should provide their own overrides.

For example, a TeachingBubble may have "normal" and "primary" bubbles, where one is white, one is theme colored. But this is a teaching bubble nuance, and has nothing to do with contextual menu which also uses callout.